src/routes.tsx

```diff
@@ -308,15 +311,15 @@ export const routes: Route[] = [
               switch (name) {
                 case 'contents':
                   return {
-                    baseUrl: `/${locale}/new-member/contents`,
+                    baseUrl: `/${locale}/new-member/${name}`,
```

Comment:
Smart 🧠

Reply:
hehe thanks, I can open another PR and do the same for the other flows :)